examples/word2vec/train_word2vec.py

```diff
@@ -251,10 +260,17 @@ def main():
     trainer.extend(extensions.PrintReport(
         ['epoch', 'main/loss', 'validation/main/loss']))
     trainer.extend(extensions.ProgressBar())
+
+    trainer.extend(
+        extensions.snapshot(filename='snapshot_epoch_{.updater.epoch}'),
+        trigger=(args.snapshot_interval, 'epoch'))
+
+    if args.resume is not None:
+        chainer.serializers.load_npz(args.resume, trainer)
     trainer.run()
 
     # Save the word2vec model
-    with open('word2vec.model', 'w') as f:
+    with open(os.path.join(args.out, 'word2vec.model'), 'w') as f:
```

Comment:
The inference example will not use the trained model.
https://github.com/chainer/chainer/blob/645483f9fd994f6b7dfa39f4d0c098a7177be0b3/examples/word2vec/search.py#L8